src/renderer/components/dialogs/ViewProfile.tsx

```diff
@@ -133,6 +136,10 @@ export function ViewProfileInner({
     null
   )
   const [selfChatAvatar, setSelfChatAvatar] = useState<string | null>(null)
+  const [cachedVerifier, setCachedVerifier] = useState<null | {
```

Comment:
why `cached` here? i would expect the getter/setter to be named just `verifier` and `setVerifier`.

Reply:
yeah I could remove that "cached" it's not very important.